xarray/core/computation.py

```diff
@@ -1947,7 +1947,29 @@ def polyval(
     Returns
     -------
     DataArray or Dataset
-        Evaluated polynomial.
+    Evaluated polynomial.
+
+
+    Example
```

Comment:
```suggestion
    Examples
```
Your example is formatted wrong, see other methods as a baseline on how to format.